packages/react-draggable-dialog/src/components/DraggableDialog/DraggableDialog.test.tsx

```diff
@@ -4,6 +4,10 @@ import { DraggableDialog } from './DraggableDialog';
 
 describe('DraggableDialog', () => {
   it('should render', () => {
-    render(<DraggableDialog />);
+    render(
```

Comment:
This is not supposed to be a full test, but rather a blank file ready to receive the test code. I have them done, but I'm splitting into a separate PR to make this one smaller.

Reply:
IMO when adding logic the test should be part of the PR, otherwise we are merging something that we cannot guarantee to work heh.

but I'll leave this up to your judgement , resolving

Reply:
unresolving ->

quite a lot logic was added to this PR , It would be great to add these tests as well as part of it. I dont feel comfortable to approve such amount of logic without any automatic verification that it works.